mautic/core/CVE-2020-35124.yaml

```diff
@@ -0,0 +1,14 @@
+title: Mautic core - Highly Critical - XSS vulnerability leveraged through referrers could allow un-authorized admin access
+link: https://www.mautic.org/blog/community/security-release-all-versions-mautic-prior-2-16-5-and-3-2-4
+cve: CVE-2020-35124
+branches:
+    '3.2':
+        time:     2021-01-14 17:07:00
```

Comment:
these should have a `>= 3.0.0` as well, so that they won't cover the 2.16.5 release.

Reply:
Both vulnerabilities are affecting both versions, in case that is relevant.

Reply:
but here, the constraint `<3.2.3` would mark the 2.16.5 release as affected as it matches that constraint. That's not expected to be true, as 2.16.5 is meant to patch them too.

Reply:
see #524